app/components/placements/view.rb

```diff
@@ -25,22 +25,26 @@ def placement_summaries
         placement = placement_record.is_a?(Placement) ? placement_record : placement_record.placement
         {
           trainee: trainee,
-          title: t("components.placements.placement_#{index + 1}"),
+          title: t(
+            "components.placements.placement_#{index + 1}",
+            default: "#{(index + 1).ordinalize} Placement",
+          ),
```

Comment:
Fallback just in case we have more than 5 placements (we only have strings in the locale file for placements 1-5)

Reply:
Do we even need the translation then?

Reply:
Well we get 'First placement' - 'Fifth placement' and then it's plain old '6th placement' so they serve a purpose. In practice we should never see '6th' or above.